api/src/test/java/org/openmrs/module/patientflags/FlagServiceTest.java

```diff
@@ -20,30 +28,76 @@
  */
 public class FlagServiceTest extends BaseModuleContextSensitiveTest {
 
-	// TODO: create some actual methods that test the flagging of patients?
+	protected static final String XML_DATASET_PATH = "org/openmrs/module/patientflags/include/";
+
+	private static final String TEST_DATASET_FILE = XML_DATASET_PATH + "patientflagtest-dataset.xml";
+
+	private FlagService flagService;
+
+	/**
+	 * Tests of the Flags
+	 */
+
+	@Before
+	public void initTestData() throws Exception {
+		initializeInMemoryDatabase();
+		executeDataSet(TEST_DATASET_FILE);
+		flagService = Context.getService(FlagService.class);
+		authenticate();
+	}
 	
 	/**
 	 * Tests of the generateFlagsForPatient(Patient patient) method
 	 */
 	@Test
 	public void generateFlagsForPatient_shouldAcceptNullParameter() throws Exception {

```

Comment:
Do these tests throw any exceptions?

Reply:
these are not throwing any exception

Reply:
So can we remove the `throws Exception ` ?

Reply:
I removed the `throws Exception`  from methods